app/src/main/java/comp3350/mealbuddy/objects/Macros.java

```diff
@@ -0,0 +1,18 @@
+package comp3350.mealbuddy.objects;
+
+
+import java.util.EnumMap;
+
+public class Macros {
```

Comment:
I say we should make a superclass called Nutrients, then we can extend off into Macros and Micros. It'll be easier for the future.

Also, since these are objects you shouldn't be abreviating them. Call it Macronutrient.

So, new structure should be
abstract class Nutrient
|___ Macronutrient
|___Micronutrient